tools/scripts/build_ubuntu_x64_torchscript.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
build failed with error logs:
```
 ----------install libtorch----------
--2022-08-23 17:20:50--  https://download.pytorch.org/libtorch/cu117/libtorch-shared-with-deps-1.12.0%2Bcu117.zip
正在解析主机 download.pytorch.org (download.pytorch.org)... 13.225.103.126, 13.225.103.61, 13.225.103.35, ...
正在连接 download.pytorch.org (download.pytorch.org)|13.225.103.126|:443... 已连接。
已发出 HTTP 请求，正在等待回应... 403 Forbidden
2022-08-23 17:20:51 错误 403：Forbidden。

Archive:  libtorch.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of libtorch.zip or
        libtorch.zip.zip, and cannot find libtorch.zip.ZIP, period.
download or unzip libtorch failed.
```
May use `nvcc --version` to get cuda toolkit version

Reply:
* libtorch with cuda11.7 actually not exist
* `nvcc` means I have installed cuda;  `nvidia-smi` means driver installed.  mmdeploy+libtorch does not rely on cuda installation

Reply:
done, check `nvcc` first